<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE TS>
<TS version="2.1" language="ru_RU">
<context>
    <name>QObject</name>
    <message>
        <location filename="../src/proofnetwork/errormessagesregistry.cpp" line="60"/>
        <source>Unknown error</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../src/proofnetwork/baserestapi.cpp" line="292"/>
        <source>Host %1 is unavailable. Try again later</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../src/proofnetwork/baserestapi.cpp" line="314"/>
        <source>You don&apos;t have network connection. Please, connect your device to network and try again</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../src/proofnetwork/baserestapi.cpp" line="320"/>
        <source>Your device seems to be in network without Internet connection. Please, check if Internet is accessible and try again</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../src/proofnetwork/baserestapi.cpp" line="326"/>
        <source>Host %1 not found. Please, contact IT immediately.</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../src/proofnetwork/restclient.cpp" line="772"/>
        <source>Slow network access to %1</source>
        <translation type="unfinished"></translation>
    </message>
</context>
</TS>
